src/js/lib/jquery.kazoosdk.js

```diff
@@ -350,6 +350,9 @@
 			qubicleReports: {
 				'get': { verb: 'POST', url: 'accounts/{accountId}/qubicle_reports' }
 			},
+			recipientPerformance: {
+				'get': { verb: 'GET', url: 'accounts/{accountId}/dashboards/recipient_performance' }
```

Comment:
Since this endpoint is part of the /dashboards API, it should be listed under `dashboards` as `getRecipientPerformance`.

Reply:
Right, got it!  I made the changes you requested, thank you!